Regression/WarpX-tests.ini

```diff
@@ -1726,6 +1726,7 @@ tolerance = 1.e-14
 [averaged_galilean_2d_psatd]
```

Comment:
For the `averaged_galilean_2d_psatd` and `averaged_galilean_3d_psatd`, it would maybe be better to put the option `algo.maxwell_solver=psatd` directly in the input scripts (`inputs_avg_2d`, `inputs_avg_3d`).
In particular, this would allow someone to directly reuse these scripts for tests of the averaged galilean scheme.

Reply:
good point, thanks!